src/material/datepicker/datepicker-actions.ts

```diff
@@ -26,7 +26,7 @@ import {MatDatepickerBase, MatDatepickerControl} from './datepicker-base';
   host: {'(click)': '_applySelection()'},
 })
 export class MatDatepickerApply {
-  constructor(private _datepicker: MatDatepickerBase<MatDatepickerControl<unknown>, unknown>) {}
+  constructor(private _datepicker: MatDatepickerBase<MatDatepickerControl<any>, unknown>) {}
```

Comment:
Had to go for `any` here, because 4.7 infers it as `unknown` while 4.8 infers it as `{}` and we need to support both.